elasticgraph/lib/elastic_graph/project_template/lib/app_name/shared_factories.rb

```diff
@@ -1,6 +1,6 @@
-require "date"
 require "factory_bot"
 require "faker"
+require "yaml"
```

Comment:
I noticed this file uses `YAML` later on but wasn't requiring it. It must already be loaded in the process, but it's good to not depend on it.